### PR TITLE
CKS-233 Update 404 page

### PR DESCRIPTION
**Pull request by 1996dylanriley**
https://nicedigital.atlassian.net/browse/CKS-233

### CODE REVIEW

File: gatsby/src/pages/404.tsx [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
 
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
-			<h1>Page not found</h1>
+			<h1>We can't find this page</h1>
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:10]: Can you put the h1/lead p in a page header component from the design system as per other pages please?

File: gatsby/src/pages/404.tsx [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+			<p>Check that the web address has been typed correctly.</p>
+			You can also try:
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:12]: Can you make this a paragraph please?

File: gatsby/src/pages/404.tsx [hunk 3]
```diff
@@ -1,12 +1,26 @@
...
+			<ul>
+				<li>Look for it using the search box</li>
+				<li>
+					Browse for it from the <Link to="/">CKS homepage.</Link>
+				</li>
+			</ul>
+			<h2>Contact us</h2>
+			<a href="https://www.nice.org.uk/get-involved/contact-us">
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:20]: Could you wrap this link and the following "if you think there is a problem." in a `p`?

File: gatsby/src/pages/404.tsx [hunk 4]
```diff
@@ -1,12 +1,26 @@
...
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
-			<h1>Page not found</h1>
+			<h1>We can't find this page</h1>
+			<p>Check that the web address has been typed correctly.</p>
+			You can also try:
+			<ul>
+				<li>Look for it using the search box</li>
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:14]: Can you lowercase the l and change to looking as per the copy?
```suggestion
				<li>looking for it using the search box</li>
```

File: gatsby/src/pages/404.tsx [hunk 5]
```diff
@@ -1,12 +1,26 @@
...
+				<li>
+					Browse for it from the <Link to="/">CKS homepage.</Link>
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:16]: Can you use lowercase b, change to browsing and moving the full stop outside the link
```suggestion
					browsing for it from the <Link to="/">CKS homepage</Link>.
```

File: gatsby/src/pages/404.tsx [hunk 6]
```diff
@@ -1,12 +1,26 @@
 import React from "react";
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
 
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:9]: [After talking with Nick](https://nicedigital.atlassian.net/wiki/spaces/CKS/pages/1430388923/CKS+404+page?focusedCommentId=1465516083#comment-1465516083), we decided the page should have a breadcrumb of NICE -> CKS -> Page not found to make it clear you're within CKS. Please can you add the breadcrumbs and we'll udpate the card?

File: gatsby/src/pages/404.tsx (hunk 1 above)
Comment (1996dylanriley) [reply gatsby/src/pages/404.tsx]: Should it also be a grey panel as with the rest of NICE? Like so?
image

File: gatsby/src/pages/404.tsx [hunk 7]
```diff
@@ -1,12 +1,39 @@
...
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
 
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
-			<h1>Page not found</h1>
+			<Breadcrumbs>
+				<Breadcrumb to="https://www.nice.org.uk/">NICE</Breadcrumb>
+				<Breadcrumb to="/">CKS</Breadcrumb>
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:13]: Can you use a Link for the elementType prop please so it navigates client side?
```suggestion
				<Breadcrumb to="/" elementType={Link}>CKS</Breadcrumb>
```

File: gatsby/src/pages/404.tsx [hunk 8]
```diff
@@ -1,12 +1,39 @@
...
+				<Breadcrumb>Page not found</Breadcrumb>
+			</Breadcrumbs>
+			<header role="banner" className="page-header">
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:16]: Can you just use the react component like this please?

https://github.com/nice-digital/cks-gatsby/blob/d40489b41c727d07c57a20862b861a7daf513be9/gatsby/src/templates/Topic/Topic.tsx#L43-L46

File: gatsby/src/pages/404.tsx [hunk 9]
```diff
@@ -1,12 +1,39 @@
 import React from "react";
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
```
Comment (ediblecode) [inline gatsby/src/pages/404.tsx:4]: Nitpicking but typically we try to orgainise imports so 3rd party deps are first, then NICE imports from npm then relative imports. Don't have a linting rule for this so it's more convention, and like I say, proper nitpicking so don't worry too much about it!

**Pull request merged**